lib/scarpe/wv/shape_helper.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ShapeHelper
+    def path_commands
+      $path_commands ||= []
+    end
+  
+    def move_to(x, y)
+      validate_coordinates(x, y)
+      path_commands << "M #{x} #{y}"
+    end
+  
+    def line_to(x, y)
+      validate_coordinates(x, y)
+      path_commands << "L #{x} #{y}"
+    end
+  
+    def shape_path
+      path_commands_str = path_commands.join(" ")
+      check_path_commands(path_commands_str)
+      path_commands_str
+    end
+  
+    private
+  
+    def validate_coordinates(x, y)
+      raise ArgumentError, "Invalid coordinates: x=#{x}, y=#{y}" unless valid_coordinate?(x) && valid_coordinate?(y)
+    end
+  
+    def valid_coordinate?(coordinate)
+      coordinate.is_a?(Numeric)
+    end
+  
+    def check_path_commands(path_commands_str)
+      if path_commands_str.empty?
+        puts "ah oh Looks like something is wrong. No path commands found."
```

Comment:
Should this be an error or return a boolean or something? Just printing to console is easy to get lost.

Reply:
hm i added tht just incase , not an error as of now but might be

Reply:
ok i removed this method